### PR TITLE
Add EOY2020 fundraising banner [fix #9765]

### LINKED ISSUE #9765: Add EOY 2020 fundraising banner
## Description

We should run the usual MoFo fundraising banner for the last week of the year, **from December 21 to January 4**. 

We should be able to reuse the same banner from last year, though possibly with some copy tweaks. Last year's banner was translated in-house in French and German, if we revise the copy we may need to revise the translations as well, or let the old text remain for those locales. We could also open this up to community l10n (with a conditional in the template so the banner is only visible to locales that have translated it).

This might interfere with the current banner promoting Firefox for Android, but maybe it's time to remove that banner anyway. Or we can see what happens with two banners.

We'll want to show the fundraising banner on at least the home page but possibly on a few other pages as well. We should come up with a list of pages.

---

#### Success Criteria
- [ ] An element contributing to the definition of done for the card
- [ ] Tasks should be represented here as something like "Approved designs attached to this issue"

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds a MoFo fundraising banner to the home page (maybe more), to run from 21 December to 4 January.

## Issue / Bugzilla link
#9765 

## Testing
https://www-demo-pr-9772.herokuapp.com

### CODE REVIEW

File: l10n/en/banners/fundraising.ftl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+banner-fundraising-title = The future is being decided online. Help { -brand-name-mozilla } fight for a web for all of us.
```
Comment (alexgibson) [inline l10n/en/banners/fundraising.ftl:5]: Nit insert the URL of the home page here so localizers have a link to preview their work?

File: l10n/en/banners/fundraising.ftl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+banner-fundraising-body = Our personal data is collected every time we access the web, and it's not easy to find out what happens with it. Misinformation is making it harder for us to find the truth. Web-connected devices are pushed to market without meeting minimum security standards. The non-profit { -brand-name-mozilla-foundation } fights to make the internet safe, open and accessible to all. <strong>Will you donate today?</strong>
+banner-fundraising-monthly = Monthly
+banner-fundraising-one-time = One-time
+banner-fundraising-donate = Donate
+
+# Variables:
+#   $amount (number) - Inserts a number in US dollars, e.g. '$50'. Adapt the string in your translation for your locale conventions if needed, ex: { $amount } US$
+# You can also localize the currency (e.g. to use € instead of $), but only if your currency is available on https://donate.mozilla.org (check the currency dropdown)
```
Comment (alexgibson) [inline l10n/en/banners/fundraising.ftl:13]: Nit: Suggest keeping this comment all on one line since it's related to the variable?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of minor suggestions in the FTL file, but otherwise this looks good to me. r+wc 💸

File: l10n/en/banners/fundraising.ftl (hunk 2 above)
Comment (alexgibson) [reply l10n/en/banners/fundraising.ftl:13]: Actually, technically you're correct it's not related to the variable but to the surrounding string instead. So maybe this does make the most sense already 👍

File: l10n/en/banners/fundraising.ftl [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/accounts/
```
Comment (alexgibson) [inline l10n/en/banners/fundraising.ftl:5]: Copy pasta on the URL?

```suggestion
### URL: https://www-dev.allizom.org/
```

**Review by alexgibson — CHANGES_REQUESTED** [review]

File: l10n/en/banners/fundraising.ftl (hunk 3 above)
Comment (craigcook) [reply l10n/en/banners/fundraising.ftl:5]: 🤦 pasted from another file and didn't actually fix it. I'm an idiot.

**Review by alexgibson — APPROVED** [review]

**Pull request merged**